tests/baselines/reference/typeUsedAsTypeLiteralIndex.errors.txt

```diff
@@ -0,0 +1,54 @@
+tests/cases/compiler/typeUsedAsTypeLiteralIndex.ts(8,5): error TS1170: A computed property name in a type literal must refer to an expression whose type is a literal type or a 'unique symbol' type.
+tests/cases/compiler/typeUsedAsTypeLiteralIndex.ts(8,6): error TS2690: 'K2' only refers to a type, but is being used as a value here. Did you mean to use 'K in K2'?
+tests/cases/compiler/typeUsedAsTypeLiteralIndex.ts(11,6): error TS2300: Duplicate identifier 'K3'.
+tests/cases/compiler/typeUsedAsTypeLiteralIndex.ts(12,6): error TS2300: Duplicate identifier 'T3'.
+tests/cases/compiler/typeUsedAsTypeLiteralIndex.ts(13,5): error TS1170: A computed property name in a type literal must refer to an expression whose type is a literal type or a 'unique symbol' type.
+tests/cases/compiler/typeUsedAsTypeLiteralIndex.ts(13,6): error TS2690: 'K3' only refers to a type, but is being used as a value here. Did you mean to use 'K in K3'?
+tests/cases/compiler/typeUsedAsTypeLiteralIndex.ts(16,6): error TS2300: Duplicate identifier 'K3'.
+tests/cases/compiler/typeUsedAsTypeLiteralIndex.ts(17,6): error TS2300: Duplicate identifier 'T3'.
+tests/cases/compiler/typeUsedAsTypeLiteralIndex.ts(18,5): error TS1170: A computed property name in a type literal must refer to an expression whose type is a literal type or a 'unique symbol' type.
+tests/cases/compiler/typeUsedAsTypeLiteralIndex.ts(18,6): error TS2693: 'K3' only refers to a type, but is being used as a value here.
+
+
+==== tests/cases/compiler/typeUsedAsTypeLiteralIndex.ts (10 errors) ====
+    const K1 = Symbol();
+    type T1 = {
+        [K1]: number;
+    }
+    
+    type K2 = "x" | "y";
+    type T2 = {
+        [K2]: number;
+        ~~~~
+!!! error TS1170: A computed property name in a type literal must refer to an expression whose type is a literal type or a 'unique symbol' type.
+         ~~
+!!! error TS2690: 'K2' only refers to a type, but is being used as a value here. Did you mean to use 'K in K2'?
+    }
+    
+    type K3 = number | string;
+         ~~
+!!! error TS2300: Duplicate identifier 'K3'.
+    type T3 = {
+         ~~
+!!! error TS2300: Duplicate identifier 'T3'.
+        [K3]: number;
+        ~~~~
+!!! error TS1170: A computed property name in a type literal must refer to an expression whose type is a literal type or a 'unique symbol' type.
+         ~~
+!!! error TS2690: 'K3' only refers to a type, but is being used as a value here. Did you mean to use 'K in K3'?
+    }
+    
+    type K3 = number | string;
+         ~~
+!!! error TS2300: Duplicate identifier 'K3'.
+    type T3 = {
+         ~~
+!!! error TS2300: Duplicate identifier 'T3'.

```

Comment:
Did you really mean to name these `T3` and `K3`?

Reply:
Yeah. It is a typo.